src/lib/comms/sol-lwm2m-client.c

```diff
@@ -2932,6 +3016,29 @@ sol_lwm2m_client_notify(struct sol_lwm2m_client *client, const char **paths)
     struct sol_ptr_vector already_sent = SOL_PTR_VECTOR_INIT;
     int r;
 
+#define SOL_COAP_NOTIFY_BY_CALLBACK_ALL(__resource) \
```

Comment:
I should actually use it in here. And the other `SOL_COAP_NOTIFY_BY_CALLBACK_INT` is only used inside `dispatch_notification()`, so I will move it there as well instead of being global.